### PR TITLE
feat(Text Classifier Node): Add output fixing parser

### PR DESCRIPTION
**Pull request by burivuhster**
## Summary

This PR adds an option to enable auto-fixing model's output using `OutputFixingParser`. The auto-fixing is enabled by default.

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/AI-281/text-classifier-unexpected-non-whitespace-character

https://github.com/n8n-io/n8n-docs/pull/2451

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [x] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/@n8n/nodes-langchain/nodes/chains/TextClassifier/TextClassifier.node.ts [hunk 1]
```diff
@@ -151,6 +151,13 @@ export class TextClassifier implements INodeType {
 							rows: 6,
 						},
 					},
+					{
+						displayName: 'Enable Auto-Fixing',
+						name: 'enableAutoFixing',
+						type: 'boolean',
+						default: true,
+						description: 'Whether to enable auto-fixing for the output parser',
```
Comment (jeanpaul) [inline packages/@n8n/nodes-langchain/nodes/chains/TextClassifier/TextClassifier.node.ts:159]: Is this enough for people to understand that we will call the original model again if there's a parsing error?

**Review by jeanpaul — CHANGES_REQUESTED** [review]
I think it looks good, but one quick comment about the description; we might need a bit more explanation that this will call the LLM again when the output is broken.

**Review by jeanpaul — APPROVED** [review]
Awesome!

**Comment by janober** [comment]
Got released with [`n8n@1.58.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.58.0)

**Pull request merged**